openadapt/record.py

```diff
@@ -689,6 +690,8 @@ def record(
     plot_performance(recording_timestamp, perf_q)
 
     logger.info("done")
+    remove_latest_action_event_from_recording()
```

Comment:
I was considering this, but I wasn't sure which choice would be ideal. I figured it might be more useful to have it removed upon storing it instead of every time its retrieved, however if there are benefits to removing it after retrieving from the database please let me know! It should be easy to implement the change.